packages/appstore/src/components/get-more-wallet-card/__tests__/get-more-wallet-card.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import GetMoreWalletCard from '../get-more-wallet-card';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
```

Comment:
@thisyahlen-deriv Same here, Why you need `@deriv/stores` 🤔

Reply:
rectified